app/templates/me/index.hbs

```diff
@@ -30,9 +30,9 @@
     </div>
 
     <p>
-        If you want to use package commands from the command line, you'll need a
-        <code>~/.cargo/credentials</code>, the first step to creating this
-        is to generate a new token using the button above.
+        If you want to use package commands from the command line, you will need to
+        login with <code>cargo login</code> using one of the tokens listed below. 
+        For security reasons, you should supply a token when prompted rather than passing it on the command line. 
```

Comment:
Is there a reason why it's more secure to pass the token in when prompted rather than passing it in via the command line? That's what I do :)